CHANGELOG.md

```diff
@@ -0,0 +1,122 @@
+<!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
+
+## 0.0.11
```

Comment:
Look at https://github.com/tesla-local-control/tesla_ble_mqtt_core/pull/78/files for new version proposed by Pierre.
... I will draft a single changelog in another file to be sure we align...
Can we symlink the app changelog to this one? That would be easier

Reply:
Let me know when you have this and I will pull it in. Regarding symlinking Pierre said the folder structure needs to be aligned so I will look at this on next rlease. I'll pull it in manually for now if you can tell me which one!